lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -92,6 +92,7 @@
 from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory, check_mongo_calls
 from xmodule.partitions.partitions import Group, UserPartition
+from ..config.waffle import GENERATE_GRADE_REPORT_VERIFIED_ONLY
```

Comment:
This can be moved one line below to keep import syntax consistent.

Reply:
Let run `isort` on this one.